blocks/log.py

```diff
@@ -221,6 +227,9 @@ def _to_dataframe(self):
 
 class TrainingStatus(AbstractTrainingStatus):
     """A simple training status."""
+    def __getattr__(self, key):
```

Comment:
No, that turned out to be too much and even breaks the serialzation. Will get rid of that.